rero_ils/modules/notifications/subclasses/availability.py

```diff
@@ -125,6 +114,6 @@ def get_notification_context(cls, notifications=None):
                 context['loans'].append({
                     'document': notification.document.dumps(dumper=doc_dumper),
                     'pickup_name': loc.get('pickup_name', lib.get('name')),
-                    'pickup_until': cls._availability_end_date(lib)
+                    'pickup_until': keep_until
```

Comment:
`keep_until` could be `None` here?

Reply:
yes it could in a strange and bad configured world. Possible `None` value is managed into the template.